src/dataframe.rs

```diff
@@ -164,4 +164,24 @@ impl PyDataFrame {
         let batches = wait_for_future(py, df.collect())?;
         Ok(pretty::print_batches(&batches)?)
     }
+
+    /// Calculate the union of two `DataFrame`s, preserving duplicate rows.The
+    /// two `DataFrame`s must have exactly the same schema
+    #[args(distinct = false)]
+    fn union(&self, py_df: PyDataFrame, distinct: bool) -> PyResult<Self> {
```

Comment:
It seems confusing to me to have a `distinct` parameter to this method.

Reply:
When `df.union(df,distinct=true)`, it has the same function as `df.union_distinct(df)`

Reply:
Then do we need both functions?

Reply:
User has two ways to use union distinct, if not necessary I will remove parameter district from union.